### ISSUE TITLE
Does not compile with rust 1.79

### ISSUE THREAD

**Issue by darix** [dani-garcia/vaultwarden#4649]
```
error: field `host` is never read
   --> src/auth.rs:386:9
    |
385 | pub struct ClientHeaders {
    |            ------------- field in this struct
386 |     pub host: String,
    |         ^^^^
    |
    = note: `-D dead-code` implied by `-D unused`
    = help: to override `-D unused` add `#[allow(dead_code)]`

error: field `org_id` is never read
   --> src/auth.rs:512:9
    |
506 | pub struct OrgHeaders {
    |            ---------- field in this struct
...
512 |     pub org_id: String,
    |         ^^^^^^

error: field `org_user_type` is never read
   --> src/auth.rs:652:9
    |
648 | pub struct ManagerHeaders {
    |            -------------- field in this struct
...
652 |     pub org_user_type: UserOrgType,
    |         ^^^^^^^^^^^^^

error: field `host` is never read
   --> src/auth.rs:774:9
    |
773 | pub struct OwnerHeaders {
    |            ------------ field in this struct
774 |     pub host: String,
    |         ^^^^

error: could not compile `vaultwarden` (bin "vaultwarden") due to 4 previous errors
```

**Comment by dfunkt** [comment]
Already fixed in 377969ea6769bccd008203c5464eb361f685b787

**Comment by darix** [comment]
just noticed.
